service/create/service.go

```diff
@@ -188,7 +195,10 @@ func (s *Service) Boot() {
 						return
 					}
 
-					if err := createRole(awsSession, *key.KeyMetadata.Arn, cluster.Spec.Cluster.Cluster.ID); err != nil {
+					if err := createRole(awsSession,
+						*key.KeyMetadata.Arn,
+						s.s3Bucket,
+						cluster.Spec.Cluster.Cluster.ID); err != nil {
```

Comment:
I think this is harder to read because of the missing indentation difference between the condition and the code being executed in case the condition is met. Since there is no tool/integration which enforces a specific style, go ahead with what you think is right.